modules/custom_layers/README.md

```diff
@@ -0,0 +1,46 @@
+Module with guides how to enable in Intel OpenVINO some layers from PyTorch:
+
+* [torch.fft](examples/fft)
+* [nn.functional.grid_sample](/examples/grid_sample)
+
+And other custom operations:
+
+* [calculate_grid](/examples/calculate_grid) and [sparse_conv](/examples/sparse_conv) from [Open3D](https://github.com/isl-org/Open3D)
+* [complex_mul](/examples/complex_mul) from [DIRECT](https://github.com/NKI-AI/direct)
+
+
+## OpenVINO custom CPU extensions
+
+You  need to build CPU extensions library which actually has C++ layers implementations:
+```bash
+source /opt/intel/openvino_2022/setupvars.sh
+
+cd user_ie_extensions
+mkdir build && cd build
+cmake .. -DCMAKE_BUILD_TYPE=Release && make -j$(nproc --all)
+```
+
+And add compiled extensions library to your project:
+
+```python
+from openvino.runtime import Core
+
+# Create Core and register user extension
+core = Core()
+core.add_extension('/path/to/user_ie_extensions/build/libuser_cpu_extension.so')
+
+# Load model from .onnx file directly
+model = ie.read_model('model.onnx')
+compiled_model = ie.compile_model(model, 'CPU')
+```
+
+- You also can get OpenVINO IR model with Model Optimizer, just use extra `--extension` flag to specify a path to custom extensions:
+
+```bash
+mo --input_model model.onnx --extension /path/to/user_ie_extensions/build/libuser_cpu_extension.so
+```
+
+## These layers were supported for:
+* [/examples/complex_mul](/examples/complex_mul) and [examples/fft](examples/fft) for [DIRECT](https://github.com/NKI-AI/direct)
+* [nn.functional.grid_sample](/examples/grid_sample) for [RAFT](https://github.com/princeton-vl/RAFT)
+* [calculate_grid](/examples/calculate_grid) and [sparse_conv](/examples/sparse_conv) for [Open3D](https://github.com/isl-org/Open3D)
```

Comment:
Is this a repeat of the information at the start of the README?

Reply:
There are more projects mentioned here than at the beginning, because first I briefly describe each layer, and here I write what projects they were created for. But yes, the information is repeated.

Reply:
I guess we can just remove this paragraph then...